docs/section4/4-5.md

```diff
@@ -0,0 +1,37 @@
+# ASHVAC – Rule 4-5
+
+**Rule ID:** 4-5
+ 
+**Rule Description:** HVAC fans shall remain on during occupied and unoccupied hours in spaces that have health and safety mandated minimum ventilation requirements during unoccupied hours.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 exception #2 Schedules for the proposed building  
+**Appendix G Section Reference:**  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** 
+
+1. P-PRM is modeled with a least one space with health and safety mandated minimum ventilation requirements during unoccupied hours. 
+
+**Manual Check:** None  
+**Evaluation Context:** Evaluate each data element  
+**Data Lookup:** None  
+**Function Call:** None
+
+## Rule Logic:
+- For each zone in P_RMR: `for zone_p in P_RMR...zones:`
+    - For each space in zone: `for space_p in zone_p.zones:`
+        - Check if space(s) is of type that has health and safety mandated minimum ventilation requirements during unoccupied hours: `if space_p.lighting_space_type in [CORRIDOR_HOSPITAL,LABORATORY_EXCEPT_IN_OR_AS_A_CLASSROOM,HEALTHCARE_FACILITY_OPERATING_ROOM]:`
```

Comment:
I am not sure hospital corridors are subject to the exception. Did we plan to run it by Mike T? Should all lighting space types applicable to Healcare Facility be included?

Reply:
I emailed him and he never responded. I do not think all healthcare should be included but I don't know these rules so this opinion is of little value lol.